File: 2000번대/2500~2599/2525.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner  sc = new Scanner(System.in);
		int h = sc.nextInt();
		int m = sc.nextInt();
		int mins = sc.nextInt();
		h = (int)((mins+m)/60 + h);
		m = mins + m;
		if (m>=60) {
			m -= 60* (int)(m/60);
		}
		System.out.println(h%24+" " +m);
	}
}

File: 2000번대/2500~2599/2525.c
#include <stdio.h>
int main() {
	int h;
	int m;
	int mins;
	scanf("%d %d %d", &h, &m, &mins);
	h = (int)((mins + m) / 60 + h);
	m = mins + m;
	if (m >= 60) {
		m -= 60 * (int)(m / 60);
	}
	printf("%d %d", h % 24, m);

}
